app/Http/Controllers/DepositsController.php

```diff
@@ -35,7 +40,34 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        $data = json_decode($request->bank_account);
+        $id = $data[0]->value;
+        $coa = ChartOfAccounts::find($id);
+        $deposits = Deposits::create([
+            'chart_of_account_id' => $coa->id,
+            'status' => 'Deposited',
+            'deposit_ticket_date' => $request->deposit_ticket_date,
+            'total_amount' => $request->total_amount,
+            'remark' => $request->remark,
+        ]);
+        for($i = 0; $i < count($request->is_deposited); $i++)
+        {
+            // find the receipt reference using is_deposited value (receipt_reference_id)
+            $receipts = ReceiptReferences::find($request->is_deposited[$i]);
+            // add to coa balance
+            $coa->current_balance += $receipts->receipt->total_amount_received;
+            $coa->save();
```

Comment:
Temporary disable this part first. This is supposed to deduct from `Cash on Hand`